user_verification/serverless.yml

```diff
@@ -0,0 +1,124 @@
+plugins:
+  - serverless-offline
+  - serverless-python-requirements
+  - serverless-plugin-tracing
+  - serverless-prune-plugin
+
+service: user_verification
```

Comment:
rename it to verification, we will use the same service for org verification in future